### PR TITLE
Add kubelet config reference

### PR DESCRIPTION
**Pull request by tengqm**
This is a reference for kubelet config generated from
github.com/tengqm/genref tool using the following command:

```
./genref -format markdown -include kubelet-config
```

### CODE REVIEW

**Comment by tengqm** [comment]
Preview: https://deploy-preview-25094--kubernetes-io-master-staging.netlify.app/docs/reference/command-line-tools-reference/kubelet-config.v1beta1/

**Comment by kbhawkey** [comment]
👋  @tengqm .
I took a quick look at
https://deploy-preview-25094--kubernetes-io-master-staging.netlify.app/docs/reference/command-line-tools-reference/kubelet-config.v1beta1/#kubelet

I noticed that the in page links do not work? I tried to use the right hand TOC.
The anchors point to the same link?

**Comment by tengqm** [comment]
@kbhawkey You are right. It looks like the anchor cannot be something like "#foo.bar.gar". Let me try "#foo-bar-gar" instead.

File: content/en/docs/reference/command-line-tools-reference/kubelet-config.v1beta1.md [hunk 1]
```diff
@@ -0,0 +1,1552 @@
...
+managers (secret, configmap) are discovering object changes.</p>
+
+
+
+    
+  
+  
+    
+
+### `LoggingConfiguration`     {#LoggingConfiguration}
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/kubelet-config.v1beta1.md:1524]: I'd expect this would be more like:
```markdown
### `LoggingConfiguration`     {#kubelet-config-k8s-io-LoggingConfiguration}
```

Comment (tengqm) [reply content/en/docs/reference/command-line-tools-reference/kubelet-config.v1beta1.md:1524]: This anchor is local to the page, not a global identifier for this type. It is hard to parse types referenced from existing types: some are embedded, some are referenced, some are borrowed (from a different group version).

**Comment by tengqm** [comment]
Renamed branch, new PR coming to replace this one.
/close
